editor/app/crashes/[record_locator]/page.tsx

```diff
@@ -1,50 +1,46 @@
 "use client";
-import { useCallback } from "react";
 import { notFound } from "next/navigation";
-import Row from "react-bootstrap/Row";
+import { useCallback } from "react";
 import Col from "react-bootstrap/Col";
-import CrashMapCard from "@/components/CrashMapCard";
-import { GET_CRASH, UPDATE_CRASH } from "@/queries/crash";
-import { UPDATE_UNIT } from "@/queries/unit";
-import { UPDATE_PERSON } from "@/queries/person";
-import { useQuery } from "@/utils/graphql";
+import Row from "react-bootstrap/Row";
+
+import ChangeLog from "@/components/ChangeLog";
+import CrashDiagramCard from "@/components/CrashDiagramCard";
 import CrashHeader from "@/components/CrashHeader";
-import CrashLocationBanner from "@/components/CrashLocationBanner";
 import CrashIsTemporaryBanner from "@/components/CrashIsTemporaryBanner";
-import CrashDiagramCard from "@/components/CrashDiagramCard";
+import CrashLocationBanner from "@/components/CrashLocationBanner";
+import CrashMapCard from "@/components/CrashMapCard";
 import CrashNarrativeCard from "@/components/CrashNarrativeCard";
-import DataCard from "@/components/DataCard";
 import CrashNotesCard from "@/components/CrashNotesCard";
+import CrashRecommendationCard from "@/components/CrashRecommendationCard";
+import CrashSwapAddressButton from "@/components/CrashSwapAddressButton";
+import DataCard from "@/components/DataCard";
 import RelatedRecordTable from "@/components/RelatedRecordTable";
-import ChangeLog from "@/components/ChangeLog";
-import { crashDataCards } from "@/configs/crashDataCard";
-import { unitRelatedRecordCols } from "@/configs/unitRelatedRecordTable";
 import { chargeRelatedRecordCols } from "@/configs/chargeRelatedRecordTable";
+import { crashDataCards } from "@/configs/crashDataCard";
 import { peopleRelatedRecordCols } from "@/configs/peopleRelatedRecordTable";
+import { unitRelatedRecordCols } from "@/configs/unitRelatedRecordTable";
+import { GET_CRASH, UPDATE_CRASH } from "@/queries/crash";
+import { UPDATE_PERSON } from "@/queries/person";
+import { UPDATE_UNIT } from "@/queries/unit";
 import { Crash } from "@/types/crashes";
-import CrashRecommendationCard from "@/components/CrashRecommendationCard";
-import CrashSwapAddressButton from "@/components/CrashSwapAddressButton";
-import { useKeyboardShortcut } from "@/utils/shortcuts";
+import { useQuery } from "@/utils/graphql";
+import {
+  scrollToElementOnKeyPress,
+  useKeyboardShortcut,
+} from "@/utils/shortcuts";
```

Comment:
i just used my code editor to organize the imports on this file

Reply:
yep! i'll open an issue for it. if we can stick to using prettier plugs that'd probably be the easiest, right? since we can just update that config...

Reply:
* https://github.com/cityofaustin/atd-data-tech/issues/21419